client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -23,25 +59,24 @@ export default function Profile(): JSX.Element {
   return (
     <Grid container>
       <Paper className={classes.profileContainer}>
```

Comment:
Optional: It would be nice to have a 'Page Not Found / Something went wrong' page here

Reply:
As soon as the important tasks are done